CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- **Add:** Config for `Tokens#createToken` and `Tokens#updateToken` can now include the `allowedApplications` property.
```

Comment:
```suggestion
## 0.13.8

- **Add:** Config for `Tokens#createToken` and `Tokens#updateToken` can now include the `allowedApplications` property.
```

This version number also should be also added to package.json and package-lock.json

Reply:
Good point, I was thinking that package release is separate from the feature PR itself. I'll combine these 👍

Reply:
would be good to double-check this. Not sure, we want to release a public version just now, specially when this feature is not publicly available

Reply:
This public version needs to exists for the sake of publishing the npm package